datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -409,6 +408,37 @@ def get_metric_sources():
     return {path for path in os.listdir(get_root()) if file_exists(get_metadata_file(path))}
 
 
+def get_available_logs_integrations():
```

Comment:
Do you need tests for this?

Reply:
Not really, it's only used for our dev documentation